api/envoy/api/v2/core/config_source.proto

```diff
@@ -57,8 +57,8 @@ message AggregatedConfigSource {
 }
 
 // Configuration for :ref:`listeners <config_listeners>`, :ref:`clusters
-// <config_cluster_manager_cluster>`, :ref:`routes
-// <config_http_conn_man_route_table>`, :ref:`endpoints
+// <config_cluster_manager>`, :ref:`routes
+// <envoy_api_file_envoy/api/v2/rds.proto>`, :ref:`endpoints
```

Comment:
Probably should link to RouteConfiguration message.

Reply:
OK. I'd been thinking about which link is preferable. Then I decided to this link because the page which users visit after clicking the link should show "Routing architecture overview" link. But it's not strong opnion, so I'll fix it.

- current: https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/config_source.proto#core-configsource
- PR one: https://94844-65214191-gh.circle-artifacts.com/0/source/generated/docs/api-v2/api/v2/core/config_source.proto.html#core-configsource

Reply:
The example link you provide here is to config source, what relationship does this have with either RouteConfiguration or routing architecture overview?

Reply:
I'm sorry I didn't make it clear enough. The both pages above contain the link which goes RouteConfiguration page around "routes" word. And I wanted you to check the actual user exprerience when user clicks those links.

![image](https://user-images.githubusercontent.com/1460043/45682416-30450000-bb7b-11e8-9496-9e8c9b5af81b.png)

Anyway, I fixed those links to RouteConfiguration message and I found it works for us, so never mind.